sr_robot_launch/launch/sr_bimanual_hardware_control_loop.launch

```diff
@@ -136,6 +136,7 @@
     <rosparam command="load" file="$(arg kinematics_config_left)" ns="la_sr_ur_robot_hw"/>
     <param name="ra_sr_ur_robot_hw/speed_scale" type="double" value="$(arg right_arm_speed_scale)"/>
     <param name="la_sr_ur_robot_hw/speed_scale" type="double" value="$(arg left_arm_speed_scale)"/>
+    <node name="sr_ur_arm_helper" pkg="sr_robot_launch" type="sr_ur_release" output="screen">
```

Comment:
i really dont like the `release` bit in the name. can we change it to something more explanatory? or all lest say arm_lock_release or something?

Reply:
How's sr_ur_arm_unlock?